src/components/entity/ha-entity-humidifier.ts

```diff
@@ -0,0 +1,62 @@
+import { css, CSSResult, html, TemplateResult } from "lit-element";
+import { HaEntityToggle } from "./ha-entity-toggle";
+
+class HaEntityHumidifier extends HaEntityToggle {
+  protected render(): TemplateResult {
+    if (!this.stateObj) {
+      return super.render();
+    }
+
+    return html`
+      <div class="target">
```

Comment:
We should not show both the state and a toggle, I think we should just show the state like we do for climate devices.

Reply:
Why? It looks good and informative and convenient, from my perspective.
The climate devices don't only show the state. It shows state, hvac action, preset, current and target temperature.
My way is much more compact and integrates seamlesly to the group toggle (so you can turn off all your humidifiers at once and don't lose the ability to see the target humidity).
Please find the comparison to climate and cover devices:
![humidifier_comparison](https://user-images.githubusercontent.com/2741408/85004860-b4695480-b160-11ea-8d62-cbbaf6c3b1d8.png)

Reply:
This PR is to get initial support in and should do the bare minimum. Let's stick with a toggle and we can iterate on the row design in a different PR.

Reply:
OK, done.